### PR TITLE
Minor updates to XForms CLI code

### PR DESCRIPTION
**Pull request by ctsims**
Minor updates to XForms CLI code to allow printing out state more easily, and to integrate with the CommCare CLI push

### CODE REVIEW

File: util/schema-gen/src/org/javarosa/engine/xml/XmlUtil.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import java.io.Writer;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
```
Comment (phillipm) [inline util/schema-gen/src/org/javarosa/engine/xml/XmlUtil.java]: xml.transform.* imports are unused.

File: util/schema-gen/src/org/javarosa/engine/xml/XmlUtil.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+            DocumentBuilder db = dbf.newDocumentBuilder();
+            InputSource is = new InputSource(new StringReader(in));
+            return db.parse(is);
+        } catch (ParserConfigurationException e) {
+            throw new RuntimeException(e);
+        } catch (SAXException e) {
+            throw new RuntimeException(e);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```
Comment (phillipm) [inline util/schema-gen/src/org/javarosa/engine/xml/XmlUtil.java]: These catches have identical bodies and can hence be combined.

File: util/schema-gen/src/org/javarosa/engine/xml/XmlUtil.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+
+import org.w3c.dom.Document;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+import com.sun.org.apache.xml.internal.serialize.OutputFormat;
+import com.sun.org.apache.xml.internal.serialize.XMLSerializer;
```
Comment (phillipm) [inline util/schema-gen/src/org/javarosa/engine/xml/XmlUtil.java]: Not finding these on my local machine. Do I need to set up some libs in order to build this? If so, can you include them in the PR or give instructions?

Comment (ctsims) [inline util/schema-gen/src/org/javarosa/engine/xml/XmlUtil.java]: These standard in Java 7.

**Comment by ctsims** [comment]
@phillipm Imports fixed across the project, fixed exceptions, and responded to comments

**Pull request merged**